### PR TITLE
Fix jest issue

### PR DESCRIPTION
**Pull request by alexkeating**
## GitHub Issue

#141 

## Changes

- Change jest configuration to reflect ceramic core
- Add ceramic data to profile

## Packages Added

- @swc/jest

## Checks

Before making your PR, please check the following:

- [x] Critical lint errors are resolved
- [x] App runs locally
- [x] App builds locally (run the build command for *any impacted package* and check for any errors before the PR)

### CODE REVIEW

File: .github/workflows/ci_master.yaml [hunk 1]
```diff
@@ -24,12 +24,13 @@ jobs:
         run: |
           touch .env
           echo NX_RIVET_KEY=${{ secrets.RIVET_KEY }}  >> .env
+          echo CERAMIC_NETWORK=${{ secrets.CERAMIC_NETWORK }}  >> .env
```
Comment (jonathanprozzi) [inline .github/workflows/ci_master.yaml:27]: I'd love a brief TLDR at some point when you have bandwidth of what you had to do to fix this -- it looks like you had to add a specific CERAMIC_NETWORK key for this to work?

Mainly just curious!

File: libs/dao-data/test/profile.spec.ts [hunk 2]
```diff
@@ -22,4 +22,12 @@ describe('haus', () => {
     );
     expect(profile.ens).toBe('samkuhlmann.eth');
```
Comment (jonathanprozzi) [inline libs/dao-data/test/profile.spec.ts:23]: I wonder if we should try to set up a profile for a DAOhaus ENS or something so we don't have to use Sam's in the future?

**Review by jonathanprozzi — APPROVED** [review]
This is great -- awesome work @alexkeating 
🔥🔥🔥

File: .github/workflows/ci_master.yaml (hunk 1 above)
Comment (alexkeating) [reply .github/workflows/ci_master.yaml:27]: Yes, definitely

File: libs/dao-data/test/profile.spec.ts (hunk 2 above)
Comment (alexkeating) [reply libs/dao-data/test/profile.spec.ts:23]: Yes, agreed

**Pull request merged**